### PR TITLE
feat(pie-monorepo): DSW-000 bump some components to v1

### PR DESCRIPTION
**Pull request by jamieomaguire**
## Describe your changes (can list changeset entries if preferable)
- Bumps some components to version `1.0.0`

## Author Checklist (complete before requesting a review)
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview
- [x] If changes will affect consumers of the package, I have created a changeset entry.

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Storybook` PR preview
- [x] I have reviewed the `PIE Docs` PR preview

### Reviewer 2 @raoufswe 
- [x] I have reviewed the `PIE Storybook`/`PIE Docs` PR preview

### CODE REVIEW

**Review by siggerzz — APPROVED** [review]

**Review by raoufswe — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Comment by xander-marjoram** [comment]
This should also trigger a docs deployment because the component statuses are changing.
For now I've checked locally and the statuses do seem to have been updated, so I'm happy with this PR.

**Review by xander-marjoram — DISMISSED** [review]

File: packages/components/pie-tag/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "**/*.d.ts"
   ],
   "pieMetadata": {
-    "componentStatus": "beta"
+    "componentStatus": "stable"
```
Comment (raoufswe) [inline packages/components/pie-tag/package.json:16]: we are introducing a breaking change for the tag component on this PR
https://github.com/justeattakeaway/pie/pull/2023, should we update it there instead so we don't have to release two majors?

Comment (xander-marjoram) [reply packages/components/pie-tag/package.json:16]: Good point. I think we can make the "major" change in both places but we should make sure to release them as part of the same changeset.

Comment (xander-marjoram) [reply packages/components/pie-tag/package.json:16]: Although actually I would argue that `pie-tag` is not stable if we're things like this...

Comment (jamieomaguire) [reply packages/components/pie-tag/package.json]: great spot @raoufswe - reverted tag

**Review by raoufswe — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Pull request merged**